### ISSUE TITLE
Problems with 6 Ghz Wifi on Pixel 8 and 8a.

### ISSUE THREAD

**Issue by gsture** [GrapheneOS/os-issue-tracker#3567]
I have problems with 6Ghz wifi (6e and 7) on pixel 8 and 8a.

- The phones have trouble finding 6Ghz only networks. Sometimes the network does not show up even if the phone has connected to the network before. Then after a few minutes or so the network is visible again and the phone connects fine. After a disconnect it will again not always find the network.

- The same thing happens when roaming to another access point (new channel). The phone will disconnects from the first access point instead of (fast) roam and can not connect to the next access point because it can not find the network. Sometimes it will find the network after a few minutes and connect again.

- If i merge the 6Ghz network with 5Ghz and turn on band steering everything works fine and the phone gets pushed to 6Ghz automatically. 5Ghz only networks also work fine.

- Signal is excellent at around 60 dBm when connected.

**Comment by gsture** [comment]
> Likely not specific to GrapheneOS.

When I first got the device it had no problem connecting but did not test any further. I get another pixel 8 this week then I will test with stock os.

**Comment by gsture** [comment]
You are right, this is not a specific grapheneos bug. I observed the same issues with the stock os. 

I have set up 2 wifi networks on the same aps: 1 network with 5Ghz + 6Ghz and another network with the exact same settings but with 6ghz only. Both phones can always find the first network and they always connect to 6ghz. The 6ghz only network is just not found most of the time.

Really strange this is.

**Comment by k0styan3** [comment]
пока будет модем от самсунга - нормальной связи не будет!!

**Comment by TipsyTheCat** [comment]
i believe it's because google sends out the international model to buyers, and since 6GHz isn't legal everywhere it's disabled in the wifi chip's firmware
